hail/src/test/scala/is/hail/annotations/UnsafeSuite.scala

```diff
@@ -309,19 +311,19 @@ class UnsafeSuite extends HailSuite {
       tv.typeCheck(a2)
 
       region.clear()
-      rvb.start(tv.physicalType.fundamentalType)
+      rvb.start(t)
```

Comment:
Could you help me understand why this is no longer calling fundamentalType (i.e t.fundamentalType). Same for rvb2.start

Reply:
unnecessary:
```
  def start(newRoot: PType) {
    assert(inactive)
    root = newRoot.fundamentalType
  }
```

Reply:
Thanks!